extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -1058,19 +1058,40 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
             this.element.getAttribute('data-amp-slot-index'));
         return Promise.reject('Cannot access body of friendly frame');
       }
-      return this.attemptChangeHeight(
-          this.iframe.contentWindow.document.body./*OK*/clientHeight)
-          .then(() => {
-            this.fireFluidDelayedImpression();
-            this.reattemptToExpandFluidCreative_ = false;
-          })
-          .catch(() => {
-            this.reattemptToExpandFluidCreative_ = true;
-          });
+      return this.setCssPosition_('static').then(() => {
+        return this.attemptChangeHeight(
+            this.iframe.contentWindow.document.body./*OK*/clientHeight)
+            .then(() => {
+              this.fireFluidDelayedImpression();
+              this.reattemptToExpandFluidCreative_ = false;
+            })
+            .catch(() => {
+              this.reattemptToExpandFluidCreative_ = true;
+              this.setCssPosition_('absolute');
+            });
+      });
     }
     return Promise.resolve();
   }
 
+  /**
+   * Sets the CSS 'position' property of this.element.
+   * @param {string} position The CSS position value.
+   * @return {!Promise} A promise that resolves when mutation is complete.
+   * @private
+   */
+  setCssPosition_(position) {
+    return this.measureMutateElement(
+        /** MEASURER */ () => {
+          this.getResource().measure();
```

Comment:
Measure happens before the mutate. Is the intention here to change the style, then measure?

Reply:
Come to think of it, I'm not sure if I need `this.getResource().measure()` -- it's just something I copied & pasted. But, yes, the idea is to change the style, and then _then_ read `this.iframe.contentWindow.document.body./*OK*/clientHeight`.

Reply:
You should only need a `mutateElement`, then.

Reply:
Done.